src/vmm/src/builder.rs

```diff
@@ -943,19 +948,17 @@ pub(crate) fn attach_boot_timer_device(
 
 fn attach_entropy_device(
     vmm: &mut Vmm,
-    event_manager: &mut EventManager,
     cmdline: &mut LoaderKernelCmdline,
+    event_manager: &mut EventManager,
+    entropy_device: &Arc<Mutex<Entropy>>,
```

Comment:
Would it make sense to replace `&Arc<Mutex<Entropy>>` with `Arc<Mutex<Entropy>>`?

Reply:
The only thing this `Arc` is used for is to be `clone`d later on. So if we were just passing an `Arc` instead of an `&Arc` we would need to perform two `clone`s. Does that make sense to you?